examples/etcd/resources/monitor-job.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: appcontroller.k8s/v1alpha1
+kind: Definition
+metadata:
+  name: monitor-clustername-job
+meta:
+  timeout: 0
+job:
+  apiVersion: batch/v1
+  kind: Job
```

Comment:
I get 2 things from a job:
1) Since it is run until completion, if the node on which the job's pod is running fails, the job will restart the pod on another node. This is an HA requirement. Similar behavior can be achieved with replication controller (-set, deployment) with replica count =1, but job seems more natural solution since there is no real replication
2) If one decides not to have automatic monitoring and do manual node recovery or utilize some other solution, all he has to do is to delete the job which will delete the pod automatically. With replication controller you need to remember to delete it first and only then to delete the pod